src/main/java/edu/kis/vh/nursery/collection/IntArrayStack.java

```diff
@@ -0,0 +1,59 @@
+package edu.kis.vh.nursery.collection;
+
+public class IntArrayStack implements StackImplementation {
+	
+	private static final int IS_FULL_RHYMER = 11;
+	
+	private int[] rhymerArray = new int[STACK_CAPACITY];
```

Comment:
Stack capacity is a property only of this implementation (should remain inside this class)